code/harry_potter_store/test/test_store.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+
+from harry_potter_store.src.store import Store
+
+
+class TestMain(unittest.TestCase):
+
+    def test_can_create_store(self):
+        Store()
+
+    def test_store_contain_five_book_types(self):
+        self.assertEqual(len(Store.book_types), 5)
```

Comment:
Just FYI typical convention for assert arguments is `expected, actual` and not the other way around :)